onnx/defs/quantization/defs.cc

```diff
@@ -9,11 +9,12 @@ namespace ONNX_NAMESPACE {
 
 static const char* QuantizeLinear_ver21_doc = R"DOC(
 The linear quantization operator. It consumes a high precision tensor, a scale, and a zero point to compute the low precision / quantized tensor.
-The scale factor and zero point must have same shape, and can be either a scalar for per-tensor / per layer quantization, or a 1-D tensor for per-axis quantization.
+The scale factor and zero point must have same shape, and can be either a scalar for per-tensor / per layer quantization, a 1-D tensor for per-axis quantization,
+or have identical rank to the input for blocked quantization.
 The quantization formula is `y = saturate ((x / y_scale) + y_zero_point)`.
 For saturation, it saturates to [0, 255] if it's uint8, [-128, 127] if it's int8, [0, 65535] if it's uint16, or [-32768, 32767] if it's int16.
 For (x / y_scale), it's rounding to the nearest even. Refer to https://en.wikipedia.org/wiki/Rounding for details.
-'y_zero_point' and 'y' must have same type.
+'y_zero_point' and 'y' must have the same type.
 'y_zero_point' is usually not used for quantization to float8e4m3fn, float8e4m3fnuz, float8e5m2, float8e5m2fnuz,
 but the quantization formula remains the same for consistency and
 the type of the attribute 'y_zero_point' still determines the quantization type.
```

Comment:
```suggestion
`y_zero_point` is usually not used for quantization to `float8e4m3fn`, `float8e4m3fnuz`, `float8e5m2`, `float8e5m2fnuz`,
but the quantization formula remains the same for consistency and
the type of the attribute `y_zero_point` still determines the quantization type.
```

Reply:
Fixed as part of rewriting of the quantization operator definition.